### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by xMcouro** [THEREALWWEFAN231/Inertia#713]
**Minecraft Version(s)** 1.16.2

**What is the suggestion** (Live) MIDI Support to Note Bot. This would include an instrument mapping to the different octaves of the piano, being the left parts double bass instrument and the right part bell, to give an example.

**Why would this feature be useful** You would be able to play thru your piano, or also play any song without the need of importing it into a .nbs song. So you can play any song suggestions from external players without making them wait to do the process of .midi to .nbs file transfering and then putting the file into the notebot folder.

**Comment by MLGgamingpro** [comment]
This is extremely unnecessary, i would love to see it implemented, but probably not possible, unless you bind 25 keys (5x5) each to play a note, and use your piano though that.

**Comment by xMcouro** [comment]
For the piano part especially, i found this very interesting. Theres NO way nowadays to play your piano live in any server. This would be revolutionary in that case, as it wouldn't need the server to have any plugin for you to play. You play directly to the noteblocks.

**Comment by MLGgamingpro** [comment]
As i said, if you could bind each note to a specific key, use the piano's keys or a AHK script i could see this being implemented.

**Comment by xMcouro** [comment]
Yes. Would be very fun, you could also actually let other people play on it, by inputting the midi from multiplayerpiano site and anyone could play in it if they didnt want to install anything.

**Comment by Cypphi** [comment]
> 
> 
> Yes. Would be very fun, you could also actually let other people play on it, by inputting the midi from multiplayerpiano site and anyone could play in it if they didnt want to install anything.

this is one of the dumbest ideas i've seen so far, but I love it lmao, it would be so nice to use a midi keyboard with notebot or some shit. That would be so funny, showing off keyboard skills via minecraft
